src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -2584,7 +2589,6 @@
 				B1FC2D1C24D9C87F00083C81 /* DMKeysViewController.swift */,
 				B16457B424DC11EF002879EB /* DMLastSubmissionRequetViewController.swift */,
 				B16457BA24DC3309002879EB /* DMLogsViewController.swift */,
-				EB6B5D872539AE9400B0ED57 /* DMNotificationsViewController.swift */,
```

Comment:
Not sure what happened here! 😳

Reply:
Maybe some old merge conflict trash that got cleaned up?